tests/array/bad/type_error_in_filter.catala_en

```diff
@@ -31,11 +31,11 @@ scope Test:
 
 ```catala-test-cli
 $ catala dcalc
-┌─[ERROR]─ 1/2 ─
+┌─[ERROR]─
 │
 │  Error during typechecking, incompatible types:
 │  ─➤ St
-│  ─➤ list of any
+│  ─➤ list of <ty1>
```

Comment:
This should read `<type variable 1>`

Reply:
Can do ; but do we wan't to expose the notion of type variables to the user ? Since the syntax is `anything of type ty`, maybe `<some type ty1>` or `<some type(1)>` would be more accessible ?

Note that here the type variable is anonymous, but if we had a named type the programmer-supplied name would be reused (within chevrons), like in https://github.com/CatalaLang/catala/pull/775/files/40e030f2efbcc08445bce227b83923b2c0bd2113#diff-66500b36753d56ce55a89631a18780eea0f413a3f63c2d753f3555ba4a34479c for example.